src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -521,25 +590,47 @@ private static IEnumerable<T> GetIEnumerableQueryParameterValue<T>(NameValueColl
                 }
             }
 
-            return results;
+            if (results.Any())
+            {
+                collection = results;
+                return true;
+            }
+
+            collection = null;

```

Comment:
I think this is actually an internal breaking change that could result in null-reference exceptions elsewhere. Previously this API never returned `null`, but now it can. I suggest you instead set `collection` to `Enumerable.Empty<T>()`.